drive/google-drive/src/main/java/com/ivy/drive/google_drive/di/GoogleDriveModuleDI.kt

```diff
@@ -12,4 +15,10 @@ import dagger.hilt.components.SingletonComponent
 abstract class GoogleDriveModuleDI {
     @Binds
     abstract fun googleDriveService(impl: GoogleDriveServiceImpl): GoogleDriveService
+
+    @Binds
+    abstract fun googleDriveInitializer(impl: GoogleDriveInitializerImpl): GoogleDriveInitializer
+
+    @Binds
+    abstract fun hasDrive(impl: GoogleDriveInitializerImpl): HasDrive
```

Comment:
I'm thinking if we can come up with a better name for `HasDrive`.

Suggestions:
- `HasDrive` -> `GoogeDriveProvider`
- `GoogleDriveInitializerImpl` -> `GoogleDriveConnection` / `GoogleDriveStore`